File: 10818.c
#include<stdio.h>
#include<stdlib.h>
int main() {
	int n, max, low;
	scanf("%d", &n);
	int* arr = NULL;
	arr = (int*)malloc(sizeof(int) * n);
	for (int i = 0; i < n; i++)
	{
		scanf("%d", &arr[i]);
	}
	max = *arr;
	low = *arr;
	for (int i = 0; i < n - 1; i++)
	{
		if (arr[i] < arr[i + 1])
		{
			if (arr[i + 1] > max) {
				max = arr[i + 1];
			}
			if (arr[i] < low) {
				low = arr[i];
			}
		}
		else {//arr[i]>arr[i+1]
			if (arr[i] > max) {
				max = arr[i];
			}
			if (arr[i + 1] < low) {
				low = arr[i + 1];
			}
		}
	}
	printf("%d %d", low, max);
	free(arr);
}

File: 10818.py
N=int(input())
int_list=list(map(int,input().split()))

max=int_list[0]
low=int_list[0]
i=1

for i in range(len(int_list)):
	if max < int_list[i]:
		max=int_list[i]
	if low > int_list[i]:
		low=int_list[i]

print(low,max)